[DOC] ModuleNotFoundErrors on initial imports in sdr_wfm_demod.ipynb

## Report incorrect documentation

**Location of incorrect documentation**
- https://github.com/rapidsai/cusignal/blob/branch-22.06/notebooks/sdr/sdr_wfm_demod.ipynb

**Describe the problems or issues found in the documentation**
On Docker `devel` images, I get ModuleNotFoundErrors when trying to import SoapySDR and pyaudio, requiring me to install them manually before running the notebook.

**Steps taken to verify documentation is incorrect**
- Pull the latest 22.04 `devel` images:

```
docker run --gpus all -p 8888:8888 -p 8787:8787 -p 8786:8786 rapidsai/rapidsai-core-dev-nightly:22.04-cuda11.5-devel-ubuntu20.04-py3.9
```

- Run the first cell in [sdr_wfm_demod.ipynb](https://github.com/rapidsai/cusignal/blob/branch-22.06/notebooks/sdr/sdr_wfm_demod.ipynb)
- Encounter ModuleNotFoundError on SoapySDR import
- Conda install SoapySDR
- Encounter ModuleNotFoundError on pyaudio import

**Suggested fix for documentation**
If possible, we should add a block to this notebook initializing the environment, which would make SoapySDR or pyaudio available for the subsequent blocks. Failing this, I think a short introduction explaining what third-party libraries are required (and the preferred method of install) would make it easier for users to get started with this notebook.

I don't think a conda install of SoapySDR is sufficient to unblock this notebook, as I later ran into runtime errors with the Conda package:

```
RuntimeError                              Traceback (most recent call last)
/tmp/ipykernel_4149/2920872161.py in <module>
      1 args = dict(driver="lime") # SDR Selection (eg.: lime, rtlsdr, plutosdr...)
----> 2 sdr = SoapySDR.Device(args)
      3 sdr.setSampleRate(SOAPY_SDR_RX, 0, samp_rate)
      4 sdr.setFrequency(SOAPY_SDR_RX, 0, fm_freq)

/opt/conda/envs/rapids/lib/python3.9/site-packages/SoapySDR.py in __new__(cls, *args, **kwargs)
   1862 class Device(Device):
   1863     def __new__(cls, *args, **kwargs):
-> 1864         return cls.make(*args, **kwargs)
   1865 
   1866 def extractBuffPointer(buff):

/opt/conda/envs/rapids/lib/python3.9/site-packages/SoapySDR.py in make(*args)
   1478     @staticmethod
   1479     def make(*args):
-> 1480         return _SoapySDR.Device_make(*args)
   1481 
   1482     @staticmethod

RuntimeError: SoapySDR::Device::make() no match
```

I also had to do a conda install of pyaudio, as attempting a pip install ran into errors on the build:

```
Collecting pyaudio
  Downloading PyAudio-0.2.11.tar.gz (37 kB)
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: pyaudio
  Building wheel for pyaudio (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [16 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.linux-x86_64-3.9
      copying src/pyaudio.py -> build/lib.linux-x86_64-3.9
      running build_ext
      building '_portaudio' extension
      creating build/temp.linux-x86_64-3.9
      creating build/temp.linux-x86_64-3.9/src
      /usr/bin/gcc -Wno-unused-result -Wsign-compare -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /opt/conda/envs/rapids/include -fPIC -O2 -isystem /opt/conda/envs/rapids/include -fPIC -I/opt/conda/envs/rapids/include/python3.9 -c src/_portaudiomodule.c -o build/temp.linux-x86_64-3.9/src/_portaudiomodule.o
      src/_portaudiomodule.c:29:10: fatal error: portaudio.h: No such file or directory
         29 | #include "portaudio.h"
            |          ^~~~~~~~~~~~~
      compilation terminated.
      error: command '/usr/bin/gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pyaudio
```

xref https://github.com/rapidsai/cusignal/issues/431, which also raised this issue - from that thread, it looked like @taureandyernv suggested doing a Conda install of SoapySDR, but after trying it out myself I'm not sure if that's the solution here

--- a/notebooks/sdr/rtlsdr_offline_demod_to_transcript.ipynb
+++ b/notebooks/sdr/rtlsdr_offline_demod_to_transcript.ipynb
@@ -21,7 +21,9 @@
    "metadata": {},
    "source": [
     "## [First-Run Only] Environment Setup\n",
-    "The below cell installs pyrtlsdr, the Nemo ASR toolkit, QuartzNet model, and required dependencies. You only need to run this once."
+    "The below cell installs pyrtlsdr, the Nemo ASR toolkit, QuartzNet model, and required dependencies. You only need to run this once.\n",
+    "\n",
+    "The use of this notebook requires an SDR device to be connected to the local machine."
    ]
   },
   {

--- a/notebooks/sdr/rtlsdr_offline_demod_to_transcript.ipynb
+++ b/notebooks/sdr/rtlsdr_offline_demod_to_transcript.ipynb
@@ -27,24 +27,25 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
    "outputs": [],
    "source": [
-    "## Install dependencies\n",
-    "!pip install wget\n",
-    "!apt-get install sox libsndfile1 ffmpeg\n",
-    "!pip install unidecode\n",
+    "# apt dependencies\n",
+    "!apt-get update && apt-get install -y sox libsndfile1 ffmpeg\n",
     "\n",
-    "## Install NeMo\n",
-    "BRANCH = 'main'\n",
-    "!python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[all]\n",
+    "# conda dependencies\n",
+    "!mamba install -y soapysdr-module-rtlsdr\n",
     "\n",
-    "## Grab the config we'll use in this example\n",
+    "# pip dependencies\n",
+    "!pip install unidecode simplesoapy\n",
+    "\n",
+    "# source install and configure NeMo\n",
+    "!python -m pip install \"git+https://github.com/NVIDIA/NeMo.git@main#egg=nemo_toolkit[all]\"\n",
     "!mkdir configs\n",
-    "!wget -P configs/ https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/examples/asr/conf/config.yaml\n",
-    "\n",
-    "## Install simplesoapy\n",
-    "!pip install simplesoapy"
+    "!wget -P configs/ \"https://raw.githubusercontent.com/NVIDIA/NeMo/main/examples/asr/conf/config.yaml\""
    ]
   },
   {
@@ -56,17 +57,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "[NeMo W 2021-12-08 11:45:33 optimizers:50] Apex was not found. Using the lamb or fused_adam optimizer will error out.\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "# CPU Powered Libraries\n",
     "import numpy as np\n",
@@ -100,19 +93,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "Found Rafael Micro R820T tuner\n",
-      "Found Rafael Micro R820T tuner\n",
-      "[R82XX] PLL not locked!\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "sdr = simplesoapy.SoapyDevice('driver=rtlsdr')\n",
     "sdr.sample_rate = 2.56e6 # Radio sample rate (Hz)\n",
@@ -138,18 +121,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "[INFO] Using format CF32.\n",
-      "Allocating 15 zero-copy buffers\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "len_collect = 15 #seconds\n",
     "\n",
@@ -175,22 +149,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAYEAAAD6CAYAAABHy/uSAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjUuMCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8/fFQqAAAACXBIWXMAAAsTAAALEwEAmpwYAAAwJUlEQVR4nO3deXhTVfoH8O/btFCWQllahJZStrLvBRQQAVnKoiiOCuOoKCMuoI7bWHAXGRj5zeioOA4q4goybqBFUBiloixFRNm1FoSy2ILsO+X9/ZGkpulNctfcJPf9PE8f6Elyc9Ik973nnPecQ8wMIYQQzhRndwWEEELYR4KAEEI4mAQBIYRwMAkCQgjhYBIEhBDCwSQICCGEg0kQEEIIB4u3uwJa1K9fnzMzM+2uhhBCRJVvv/12PzOnKN1mexAgohoA8gE8xsyfBLtvZmYm1q5dG56KCSFEjCCiXwLdprs7iIhmE1EJEW30K88hom1EVEhEuSoO9SCA+XrrIYQQQj8jLYE5AF4A8Ia3gIhcAGYCGASgGEABES0E4AIwze/xNwPoCGAzgEQD9RBCCKGT7iDAzPlElOlX3ANAITMXAQARzQMwkpmnARjhfwwi6g+gBoC2AE4S0SJmPq+3TkIIIbQxe0wgDcAun9+LAfQMdGdmfggAiGgsgP1KAYCIxgMYDwAZGRlm1lUIIRzP7BRRUigLuUwpM88JNCjMzLOYOZuZs1NSFAe3hRBC6GR2ECgG0Njn93QAe0x+DiGEECYxOwgUAGhJRE2JqAqA0QAWmvwcQgghTGIkRXQugJUAWhFRMRGNY+ZzACYCWAJgC4D5zLzJnKoKIURly38sxa7fTthdjahlJDtoTIDyRQAW6a6REEJocOPsNQCAHdOH21yT6CRrBwlhgq9+KsW5MsluFtFHgoAQBn3z835c/+oaXPz0F3ZXRQjNJAgIYdDOA+7+6L2HT9lcEyG0kyAghEEhJ8IIEcEkCAhh0IrC/XZXQQjdJAgIYdDqogN2V0EI3SQICGHQ/mNn7K6CELpJEBBCCAeTICCEEA4mQUAIExUflOULRHSRICCEARt3H67we5+/y4QxEV0kCAhhwIjnV9hdBSEMkSAghMlkDaHwYZapekZJEBDCZFM+2Wx3FRxj/a5Ddlch6kkQEEKnL7eVKJa/vvKXMNfEud5ZvdPuKkQ9CQJC6DT2tQK7q+B4//222O4qRD0JAkJY4Pjpc3ZXwXEOnZCZ23pIEBDCAu0eW1K+xLQwHzPjfb9WwNZ9R22qTXSTICCERf7x+Ta7qxCzlm4pwX3//b5C2ehZq2yqTXSTICCEDu+p6IuW7EXrHDl5VrFcZmxrJ0FACB3u97sKVXLg+Okw1MSZTpwtUyyXGdvaxdv55EQUB2AKgFoA1jLz63bWRwgzfV0o+wxY5eufZCMfs+huCRDRbCIqIaKNfuU5RLSNiAqJKDfEYUYCSANwFkDU5HodPC5ZCE72yldFqu97OEC3hTDm1DnlloDQzkh30BwAOb4FROQCMBPAUABtAYwhorZE1IGIPvH7SQXQCsBKZr4XwO0G6hI263YeRJcpn+OTH/bYXRVhk6fytqi+76z8ny2siXOdlaU5TKM7CDBzPoDf/Ip7AChk5iJmPgNgHoCRzLyBmUf4/ZTAffV/0PNYxdBOROOJaC0RrS0tLdVbXdNs2nMEALDyZ2nqi9BmfiFBwArS1WYesweG0wDs8vm92FMWyAcAhhDR8wDyle7AzLOYOZuZs1NSUsyrqV6S8hGVCkuOYvHGfXZXQ4TBkVPSBaeF2QPDpFAW8KzJzCcAjDO5DmFBSq9URKyB/3RfY2yfNgz/yS/C/qOnkTu0NeJd2q6Dys5rvwjYeeAEMupV1/w4oc+Rk2dRKzHB7mpEDbNbAsUAGvv8ng4gpjrPvaeAc2XSIohGA/+5HNM/3YpXVmzHxzrGdZpPXqT5McWHjOWuv7XqF2Tm5uE3SUhQ5cQZGTTWwuwgUACgJRE1JaIqAEYDWGjyc9jqu52HAADzCnYFv6OISD+XHi///5Mfb8aps2U4c87aQcY/vrxa82POlp3HaU8GzLuez5pMhHILNUdj8DOKPcsiAN3dQUQ0F0A/APWJqBjAY8z8KhFNBLAEgAvAbGbeZEpNI8SWvUfsroIwycETZ9H6kcUAgM/u6YusBklB76+nKyiUX4+cwpfbSnBZp0ZYtGEfzpWdx6iu6RjybD627z8e+gAOc+DYaVWztYV6uoMAM48JUL4IgPY2c5SQceHYNPiZfAxonYrt+4/ji/v7Vbp9128ncPHT+mejzvyiEBP6t6hUftNrBdi89wgefH9DednfF2/FwROVBzflswdMeGedqvv1nv4/fPXX/oiLk8G7UGTZCI1kQDh2/W9rScCrbyMBAABmLFFeTG7Xb5W7eJQCgHBbVeSfla5s96GT2CytdlUkCGgUJ1FAmOio7DtgmVdXbLe7ClFBgoBGcfIXEzZxem9QUekxTfdfsH63RTWJLXJK00hSQ4Wwx4B/LNd0fwvG8WOSBAGNInn3ImYGy+hhxDosff0iAkkQiBHMjKaTFiHXJ8tERJb8n+xf+8ppZAmJ0CQIxAjvINi7a3fJFWeEMjopzcmtvK379GX6dHz8M5wKsAGNcJMgECN8lzfuOyM2d1c6c+48MnPzMDtKsz4efP8Hu6sQtXKe/Ur3Y/+7Vmb3ByNBIAYdPnkWH6yLvVmVxzzplM//7ydLn8f/itustevPyUilLR5ZEFOLFphOgkCUW/nzAWTm5lUqv3d+6D1wI0nJUffyCcGEqzvE/2kORUj3moQQYQUJAiqdPFMWkesGjXl5ld1VMMU1L63E2NcKgt7HexIkiyfs+Z9s83+UAV291u08iF+PnDJ0jFVFsoGMlSQIqHT3vO8w9F8V+yX3Hj5pU23UyczNi5r+0B0H1K+QafWcbf8Wx30hVq0UgY168RsM/Ke2/H5/o2cZv9CRZbgDkyCgUsGOymuWPPDfH3DohH0fLqVuIH8PvBc7g5Hec/MBi7/QkdrtEq3JQUdP2b80Rtcpn9tdhYglQUAlpUW9VhTuxyUzvgx/ZQCs2a5uIa1YwmE6PUfryVYIPSQIGHT4pD2Dhtf8Z6Xq+8bKvIEjJ8NzRRmuYCNCc/LciHCRIOAAnZ78LCYCwQ/Fh8LyPFaed14wlN7qvBPiToWltvWyege5aCVBIMrsPXxS1+qInZ78zILahFe4Fu+zMgis/eWgdQePQWZ2t2Y9/Klpx4oluncWE/a4aNr/7K6CbWZ8prwxi9l8u4P2HTaW3ujvy22Sbioii7QEHORnjeuxR5rSo6fD8jy+LYFb3lgbludUw2nd41bs6SxjDJVJEFDB6GQXM5SdZ1yrYTBYyaX/WI53Vu80qUax66BP2u+G3YdtrElFqx2WEdbx8SWmH/PnUuXtQ53M1iBARBlEtJCIZhNRrp11CSbUgNL6XYcsr8Phk2dNOQlM/nADNhRHzolNrQPHKrYCCkus29fhxJnIXHVy/7HwtIQixXEL3gdZUbQy3UHAc+IuIaKNfuU5RLSNiApVnNizAOQx880A2uqti9XOh2hCXjHza8vrYOYs2cteWGHi0cKj21NLK/w+8J/5lj2XNwicOGNNSqrebQ+lJ8O4D78L/rfPzM3Dn15ZHabaqFdy5BROn7MmgBlpCcwBkONbQEQuADMBDIX7pD6GiNoSUQci+sTvJxXAdwBGE9H/AFi2/vHx0+eQ82y+rivgc2XnVWUoWN3X+M/Pf7T0+FYpO894+KMN2GUg1S/c/bhXzPwaR06dxbRFWy05/t3z1gMAPvxO20qvoS5GRGjBNp/3dvuuKNwfruqo1uNvyzD+jW8tObbuIMDM+QD8+yd6AChk5iJmPgNgHoCRzLyBmUf4/ZQAuAnAY8w8AMBwvXUJZf2uQ9i67yjunved5se2eEhdWlnTSYs0H/uXA8fLdz5aXXQg6NjDm6t+0Xz8YMK149J3Ow/irVU78Zd31+s+xu1vrVMs//j7PbqPGcrNrxWY/jf3d8+72tYkUpq1LowrO8+Y9MEP6Pm3ZeVlmbl5+HTDXhtrVdlyixYyNDtFNA2A74plxQB6Brn/YgCPE9EfAexQugMRjQcwHgAyMjJ0Veqo54RXtD9yBoUmfbABc9dUHqT95M4+aH1BEuJdv8fnbyy4Mun4+GfY9lQOqsa7TD+2L2+CR6DuLGbG0dPBu10Wb9qnWH7n3O9wWadGIeugpx/Y6nx+PXX6+Ps9eH5MFwtqEx6nzpbhjZU78DdPC+vS1ql49LK2OHb6HNo1ql3hvlZ1xSn526ItmLum8kKLt7+9Dh/c0QtdM+qErS52MHtgWOm7HrANy8wbmfkPzHwbM98f4D6zmDmbmbNTUlJMrJb51KZgMrNiAACAEc+vwIwl28rvt3jjXvzRoj7KVg8vrtDVcrbsPP78+lpk5uZhyaZ9eOWrIizSeTW059BJLNvya/mGLIGybOau2YWOj+ufyHZOxYYvkbiZS+tHFttdhbD619Kf0PqRxeUBAACWbS3BJTO+xPDnVlQaJ7FyiRDfz/z1r64O2kU06sVvDD3X97sORfxMZbODQDGAxj6/pwOwrs2uUnycviCg9Wpt+qeB+5CPnjpb/uEL9qEDgP/kF+H/lmzDwu/34LYAXSFm8dblrVW/oOVDn2Lpll8BALe++S2eytuCO95eh4U6ul1GPL8C415fix9/dWfxnA7wRfjf1l8r/H7e74QdarLWGytDd9ls22ddJlEkePC9HzDhHWs/J0Y9szT4mNbd89ZXCOhWjn/4Llv+1U+hW9l65yt8ua0EI2d+jal5m3U93suqAWEvs4NAAYCWRNSUiKoAGA1gocnPoZnLpS8I3P6WtoGYzzf/qlheWHIUHR7/DPPX7kJmbl6F/YADeeGLwvIBRCs9lbcFB4+fwcMfbQx4n7vmfocF63druqLxrt8e6i/v/133X7L7wmnLEMyTn4T+gh2xaZG/cHl37S7k/aC//3rump3IzM0r377TLt2eWoqi0mM4dbYMvaZbNzN+tcZNappPXqR5i9HNe46Ub5K02eBmVG+tsnZuj5EU0bkAVgJoRUTFRDSOmc8BmAhgCYAtAOYzs+0bfCbE6XuZX+iY4v/KV0WVygpL3N1E73+rLzXQal1UrLV+97z1lqTO+V9jlflEBbV7NRy3+eQV7Wbluz+zJTZPijx88iwG/GM57pqrPYFDi4173F2Tavbj8Fq4XltreNhzv29AVbDjoKpuy0CmqLjQMcJIdtAYZm7IzAnMnM7Mr3rKFzFzFjM3Z+ap5lVVv3idLQE9nsrbUqlLw3s9vEZhY5pooqb+hSVHscKnif34x9o+wD+XHMOyLb9iy94j6Pykuo1ABj8TfM7A92FafTQSvbN6JzbtCc/kQGY2ZZOlzwK0qM3y1qqdmgIAoH53ubLzjA6PVZ7p7B3ni0SOWEDOd0zglwPH0aReDUufb0XhfvTN+n0QO5byu5k56B6/Widx+c8BeGSB9obj7kPBt/mM1Q14bpi9BmtDBObJH24AAOyYHjgD26x5GG+t3olHPtqIpfdeghapNTV3oUS6U2fLkJgQPJuu+9Slitlu/8kvQlaDJFzVLd2q6unmiLWD4nyCgNqlaY1ML/fvW339mx26jxVpmk5aZPrKmmbIfT/wNporY3Sj8vwfS4MuceG/1Ia/Bet3IzM3r3ygdNMeY33Xy7eVAAC2e1KxrezXt0PrRxbjzrnfBRwoLio9FnQv40jdq9oRQUDPSctICt8db6+r8EGJhD1WzTTt09AD20qUAqtZbaR5BZXzvMufI3YaYppSdvcc+v1zr3S1/9CHFZMB7pz7Hb41YX6E97nCteprOH38/R68s7pyRhozY9S/Q6eTat3wfus+Y4FZDUcEgWN+J+FQs2Wf+Nj4WHawbJtot2D9Hl3barZ+ZDG+2FZSoY/a6vX1rViO2E53vK0+FfQ2n+y2G2avqfS5V8oGukrFiSwwd4ubAbz2dfA06Gh2yDNzm5nLA979//2hvDyYrlM+x1c/qf/M5zz7Veg7GeSIIOCfpxhqctJrX+8w/JRz1+zE4o3uma5GU8QiUacn9E3wuum1Agx/boWhtYQCyfebVn/izDk8GKSbKJatLjpQYazkq5/2V/jcB1v08OnFW3UtK+IdKnroww14QmNCQDQp8LSWuk75HH3+/gX2HT6F99epXwfq+lfXqLrf3sPBx7rM4owgoGDov6yPsO8WaM9CiCYvLf9Z92MPmpBF4u+G2WsqTD5r++gSvPettkXaosG4OQVBbz9/nnHtrFWKt7216hfk/1gadPnzF7/8GRPf0Z+muf+Y+e9tJMn/sRSZuXk4eOIsdh86GXIui17X/kf5PTSbI4KAUi7LljBcneuZZxBNpn+6NeTgYyBW9dPfPCdydgKzyrKtJUHTPh9eELgr8uGPNuKG2aGvRPN/LMW6nb+PD5QcOYXM3Dxk5ubh/gADnOFLxI5+mbl5eHzhpqDdqjstaC0rcUSKaLCURn+h0g1FRd2eWho0/TAQK3vqfzt+JuTcgWg3/DnlPSFmr9hu2u5xgdbNee/bYkwe1gZ1a1SpUK7hayYAzPlmB+Z8swOf39MXZ8sYbRomlZ+rwrFRlZczgkCA8rLzDJffukJ3RvgaLLFi98GTaJScaMmxu6qYAR2Lwtn12HXK51g9+VI0qPX7e1gkWzfqMshzwTKwTQO8cmM23ly5Q9d8Gb2c0R0UIAo0n1xxD4BzZeexbuch6ysUY7pPXRpw3aRAJryzDj2mWtOXKsKj59+Wlc+BuXH2GvxUom4VXaFs6ZZfkZmbF9YAADgkCMSpbKceivGFxqxSevQ0bnkj9vviRWWPLdyEzNw8yzY8EdZzRBAIFgN8UxX7q5xNLJTFciaUELHKEUEgmIuf/gKHPZM8Qu1wJYQQscbxQQAAbnlTujKEEM7kiCAQKkV0zfbf0PHxysu/CiFErHNGEFBxnyMxtsibEEKo4YwgIJNYhBBCkSOCgNoUUSGEcBpHBAEhhBDKHBEEpB0ghBDKnBEEJAoIIYSisAUBImpGRK8S0Xs+ZTWI6HUiepmIrrPw2a07tBBCRDFVQYCIZhNRCRFt9CvPIaJtRFRIRLnBjsHMRcw8zq94FID3mPkWAJdrqrkG0hIQQghlapeSngPgBQBveAuIyAVgJoBBAIoBFBDRQgAuANP8Hn8zM5coHDcdwAbP/yvvQm4SiQFCCKFMVRBg5nwiyvQr7gGgkJmLAICI5gEYyczTAIxQ+fzFcAeC9bCwa0rLpjJCCOEkRk68aQB2+fxe7ClTRET1iOglAF2IaJKn+AMAVxHRvwF8HOBx44loLRGtLS3Vt1ythAAhhFBmZGcxpXNrwF0DmfkAgNv8yo4DuCnYkzDzLACzACA7O1vXroTSEBBCCGVGWgLFABr7/J4OYI+x6lhDgoAQQigzEgQKALQkoqZEVAXAaAALzamWuUg6hIQQQpHaFNG5AFYCaEVExUQ0jpnPAZgIYAmALQDmM3N4N8dUS2KAEEIoUpsdNCZA+SIAi5RuE0IIEfkcsWyErCIqhBDKHBEEJAQIIYQyZwQBiQJCCKHIEUFACCGEMkcEARkTEEIIZY4IAhIChBBCmSOCgBBCCGXOCALSFBBCCEWOCAKybIQQQihzRhCQGCCEEIocEQSEEEIoc0QQkIaAEEIoc0YQkP4gYbGn/9DR7ioIoYsjgoAQVktNqoorOjeyuxpCaOaIICANAREO2Zl17a6CEJo5IwjYXQER8xjAdT0z8Oa4HnZXRQhNHBEEhAgHIkJ6nep2V8NxxvbKtLsKUc0RQUC6g4SIXVdnp9tdhajmiCAghIhdsiKAMQ4JAvIhEda6sGk9AEBGXekOCrekRFVbpYsAHBIEhLDOtdmNUa2KCwDgipMLjnBrLIHXkLAGASJqRkSvEtF7PmVXENHLRLSAiAaHsz5CCOF0qoMAEc0mohIi2uhXnkNE24iokIhygx2DmYuYeZxf2UfMfAuAsQCu1VB31WRgWFipaoI0qEX00tKZNgfACwDe8BYQkQvATACDABQDKCCihQBcAKb5Pf5mZi4JcvyHPccSIqo0lrRQEcVUBwFmzieiTL/iHgAKmbkIAIhoHoCRzDwNwAg1xyX3wj7TAXzKzOsUbh8PYDwAZGRkqK2uEEIIFYy2Y9MA7PL5vdhTpoiI6hHRSwC6ENEkT/GdAAYC+AMR3eb/GGaexczZzJydkpKiq5LSGySEEMqM5lYpnV850J2Z+QCA2/zKngPwnMF6BCWriAoR2x4Y0gozlmyzuxpRyWhLoBhAY5/f0wHsMXhMIaJKywY17a6C41WNl8F5vYy2BAoAtCSipgB2AxgN4I+GayVEFBjTozHG922OpvVr2F0VIXTTkiI6F8BKAK2IqJiIxjHzOQATASwBsAXAfGbeZE1VhbDfpa1Ty///0PC2EgBsVkVaAIZpyQ4aE6B8EYBFptXIAjIiIMySnVkXy7a6M51rVpXlCux0RedGeHZ0FwBAn5b1ba5N9JIwKoQOt17SzO4qCB+tL6hldxWiliOCgCQHCbtkN6ljdxWECMoZQUA6hIQQQpEjgoAQIjY0q18Dz1zbye5qxBQJAkKIqFE1wWV3FWKOBAEhLHT9RU3sroIQQTkiCMjAsLBLv1apoe8kVCMAKTUTAQCZMkfDFI4IAkLY4aepQ1G7WgL6tdK38KFQ1qdlfbw5rgcm9m9RofyNm3tg+QP97KlUFJMgIIRFElzur9es67NtrknsubhlCuJdFU9ffbNS0KSetA60ckQQaHVBkt1VEBHq9n7NLX8OWdpARDJHfDoTXI54mUKHSBkuql+zit1VEA4lZ8cYc2WXgHv6CBM0T3F3N7RqYG7rsla1BFOPJ4RaEgRizLRRHeyuQkwb3O4C5N3VR4KtTSTTz3wSBGJMokymCSl3aGtDj2/XqLbsVidihgSBGDSwTQO7qxDRkhKVl4Ae0yMj6ONGdm5kRXWERR4e3sbuKkQFCQJRbs5N3ZH/QP8KZQkuuUoNZGCbBqhfs6ribT2aBl/xs0NabSuqBAAYIJPKTCetNXUkCES5fq1SkVGveoWyKVe0V/y/AF65MRuuACcHK1ebnX/rRUFvnzRMrlqFPSQIxCDfK91AJzyhnZEry86Nk4Pe7oqT9ymYycO0j+PIX1QdCQIxLpZiwBOXt8PGJ4bg39d1Ne2YydXDk5oZ7H14dERbAMB9g7IUb39rXE8rqhRV+rVKxQ0XNcGLJr73wk2CgIgaV2eno2bVeAxqa97A9+geGUhLroa8u/qYdkytbu7TFADQtpHyFont02TrRALw5Mj2siyEBcIWBIioGRG9SkTv+ZXXIKJviWhEuOriRAPbxM7Ao/+aMUbEEeHr3AFo16g2skyeAKZVeh332E7u0Nbo0bSurXWJdhc2k7+fWqq+TUQ0m4hKiGijX3kOEW0jokIiyg12DGYuYuZxCjc9CGC++io72/CODRXLg3U3bJ82DM9c27n898EmXknHkkBX4l5jejS29PlbXZCEr/7aH7f2bRZyIFkEtumJIXgzBrvQelp0YaCcMF3ZHAAvAHjDW0BELgAzAQwCUAyggIgWAnABmOb3+JuZucT/oEQ0EMBmAImaax4BxvbKxJxvdoT1OQMN9OY/0B+/HDiheBsRVRjUjHdoCmm7ECd5wD1AW3aeFW+rXkXt16UytX/xxnWrh76TCKpGVf3vUyQb0u4CS46rqiXAzPkAfvMr7gGg0HOFfwbAPAAjmXkDM4/w+6kUADz6A7gQwB8B3EJElnVPJVnwwbgm27wrw8x6xr78jetWR5+W9U2qTWzyZk21CRIMqvnNuO7dop6ldVKDlWOSo8ifwLoLBCMn3TQAu3x+L/aUKSKiekT0EoAuRDQJAJj5IWb+C4B3ALzMzOcVHjeeiNYS0drS0lLdlb3cgtmeoboPtOjVwpoTeB+LjhvN0pKroUYVdctrmD0QKamg+ugJhLGUGQfA1IQIX0aCgNKfOOBbxcwHmPk2Zm7OzNP8bpvDzJ8EeNwsZs5m5uyUFP07NEX6gl+PXdZW1f2qalyb3nv1ILOIhRBKjASBYgC+/SHpAPYYq45zVY13Ycf04aHvl6DvLasa//uVr5UzY60UH2est3B832aaH0Nw5+l/cEcvQ88d57kszbGoX1cIvYx0lBcAaElETQHsBjAa7r59R0munoBDJ84aOkagBc2UqG0Wf3ZPXxSVHtNZo8raNaqFTXuOmHY8LZbddwmquOIM79DVW0fX2Pi+zUzpEoqLI6yZfCmSq1dB3oa9mh/fpmEtbNlrz98/VrRtWAub5W9YidoU0bkAVgJoRUTFRDSOmc8BmAhgCYAtAOYz8ybrqhqZpo/qaPgYV3VNN6EmFWU1SEJOe+V00kDjI0qD595+1cs72beCZvOUmmHLmvFvI5k5JpBaK1F3INNyoSCUvXZTd9QO8+Y9dWtE/o5xarODxjBzQ2ZOYOZ0Zn7VU76ImbM8/fxTra2qMWYO4voa0k7fYM3V3dIxqmvgcQq1A5d6BDwRRVEvUad061b0dKKGtSM7S9us7Te/f2wwPr37YlX3ffqqjoa7AVdPvtTQ48PBMctGGMnx9ufdYhBw5+BX13HCnnF1J7RvpO5EtuLB/orlXTOSNT9vrJhxdSfLnyOrQU3LnyNS9GulP+kiFKOTEzc9MQT1Aiz/HcyorukVUny9rQC1e453apyMrhnBlxePBY4JAmbKu+tibHxiiOHj1PQ08ZX2l509tjvS61QD8PtyAv4u1bl5jH8uvJdSQ6CJpxsmXOl2y+67RNX94g2mWiZ4WkMcZJBlbK+muo8faM8Cf7VUdPOE42+fUbeG4TGXQIzsnxxH+id/1a6WgLf/fGH570q77rW+wN6lQiKBBAGNumQkIzHBhZoGJp89cXk7AO6xgCcub4cJ/ZtXuk/PZvXwwR298Oa4HorHuG9QFu7oV/lxwWR4Tuhqp58/PLwNald3N8ONZBSpbbHUrVEFzVMqXn1blVc//9aLcPelLZGUWPEENazD7+MoKUnarz4BYMGE3qq7HBolVwt5n+Tq1vcr68mc0uOa7MrjXzP+0BHv/Fl5mYe6NfS9B75eG9sdS+/tq3jbuD76A32skNEmjfplGV+IzXuF74oj3NgrM+D9UpMSkZrk7qt96U9dwQysKNwPAEiuUUXz+vZL/tIXZ8rOg4hwWadG+Pj7ihm9DwxphUcWmD+2f13PJli385Dmx312T18kWzSQl9UgCVmDKl8FTr2yPe4bkoV1vxzSvehepxB7B4TLJVkpWP6jugmWrjiybVru1SbOvFfSv7V9iydGw2xvaQnYwP/qU42c9g0xtINyto9a1aq4gmZHXH9RJh4Z0VZxj2IjWQ5XdUvH0Pah8+P9u2ayGiQhtZY5A5bZTdT17ca74pCalIic9hdE/faEM67WlrnGYYgCFzYLvAzH+kcHVSqz4i0I19v60LA2lnWxmSnyaxgF3vZpynbPDH6yefG6rkGXCe7V3N61asb1aVrebQQAacnuk3C7tFpYMKG37uPG2XxCnaRjZyq7DWrbwNAmKt5WZChaZ6Eb0T7IPs3J1avgWr9WQZqK7rJIdYuni21sr0xc3c38NHCzSBBQqUGtwH2TXTLqlGchhPrQDgtxNf/aTd3x/aODtVdQoxsuaqLqfn+/qiOeH9MFrS+ohU6Nky3NItFCTV+6L7uDkB4v35Ad8vMSine8KVg6cv8wbnIfaojn73+o2HrpGKZU4Cb1qmPlpAGWHPvxy9sZzmbb/KTxRJRAJAiopHbF0Jz2DXGxgdU8q8a7UDsMWx52z6yLgocGhrxfUmICLvOZKPbSn7qpfg69qYF39GseMIPJKzHBZdn66rHkgSGtsWP6cPzzms52VwUAKg38K/n07ovDPg8ku0ldNKyt/sLi+0cHB924xqx5DV5mprj7kyCgUaALyklD26Bdo1q4uGV9Sze0uGdQFkZ0bIhRJiyIl5JUFXcNaKHpMUppdoG8eF3X8isYLWMKf81pjS1TcjTVSxgXjp3ViCjkyrZtGtbCKAtm0ZfXwYRjBLtQ+/COXvgmN/IniXlJEDBJ+7TayLvrYss3tKhfsype+GNX055ngicI6Mm7DzWZKt4VV34Fc/fAliGPl52p78p+7i0X4tYgKY4D26SincqJeZHKd0bv9mnDdB1j6pXtAQBjemRUKB/smfV+U293umSVAJOptPSo+aaC3n1p6PfeX5zn82jn0tt6JoEC7u7haBgQ9oqemgpLeHcqy6yvfY2cBRPUb86u5kvx3Ogump7fN5flvsGtAt7vlRu7R9WXUsnUKzuU/19v1tJ1PZtgx/ThmHpF+/KyL+7vV37V7T1qqwATqJbfrzxz3Z8rjsr3gX7qiva61n26JjsdN/XOxL2DsjQ/1gzv/Lmn6noPNzhuY7fo/maEkffqSO2Uc6VJMZEo3hWH2WOzMfeWC0Pf2U+1IFdKXXQsaRHseKFE+0k+lGCTEy8KknapJM7n6jpRYWnylqk1FWfEZ6jc/U7vFbSvqvEuPHZZO13p1GoF281PzSZP3Txpx/cMyipPgX7WZy/vaBHb3xwT3dK3GW7v1xw39c5Udf9HRqjbJCYSDGjdoMLs2FRPJpSRuQHB8sHNEn35PtboGWSAMpALQsy/UDMjPlAmXOMAy5xECt+W1IDWqUhLroZbL3F3J15/YRO8fEO2quPcO6gVPr+nL1qk/t4tqvYi0evrXGsykrSQGcMqJSa48GCO+lzzpMQEtE+rhY27o2/98j/3aYqGtRMNLR+tdaakVdtgBkvtdTIzJob95/puWP5jKWYs2WZCjexRr2bVCifiKT5dZaG44ggtPYPp3rii9e8aCfMgpCUgKol3xWFk57SAfc9V4+NUL5CmxqiuaXgrwNoxRt1wUaYlx7XDyM6NMFlh0ptvAG3TUNuS6b5rQnlnkzfQsKz0hP4tKm3dSgQ09LQ06nlak3fpGByOFEpdZv6idbc+wGFBIL1ONdV7+YrAtj01FGsfDj3HQJjrX6O7YHzfyosG+uaQT+yvLeXX16VtUvGv0Z1xz8DAg7EfKcwaV8qJv71fc7x4XVfkePrK7x2UpWr71HBSe82+/IH+ISe56THrevVzbqzkqCCw4sEB5Wlwwhq3XGzf3/ftP/dEy1Tn7AHg5d9gG9MjI2Sfv1J3HRFhZOe0oIPsnRsno53fBk33DW5VqQUS74rDsA4NI3L9Ja01alArUdMkSTWmjeqAwSr2m/7H1Z2w1eI5M44KAqH8NHWo6vXs1TA65T8aeecvaOkbNdqU9j5X7xb1K6W6RuA5yHS+W092yUjGtFEdsErljlZ6/j4DPKtyepMJEhNcFbLhwpEU4BRJifGaJmjqIUHAR4IrDs1TauK7RyqvZqjH7Zc0L0+X69w42daJL+FSfkL3iwHeFNtBBneZqvBcMfDnNOMz0bB2NXw0oTe2TsnRvKaSHvcMzELBQwPRwKe14bvo4KShwRMohqhYUdZK3s+Ny4IPkNlLR8e7rP+QS3YQ3Gla9XzSIevUqIIP7uiFbwr3o03DWjh2+pyu47q3nozHiTNlmHVDN9WrOkaTycNaY/HGfeX7BQT6XiUmuLD03r5IS66ONo8uDl8FI9DrN/dAneoJWLxxX6XZu3p11riHQXqdaig5ejrkTPHOjZOxftehCmVxcVRpwx3fXe7iQ6RJ/qlnBq7qmoa2jy7RVGezZNStjtsuaa5pLk/I4KriXP3stZ3xl3fXq35OwJz9S0KRIADlNK2uGXXM3V80CjaX0GN83+YY37c5sh76FGfKzge9b4tUdzrd01d1xF/f/6G83MwLMv9DReLqoZdkuVdi7ZiebFsdXr4hG6u3/xZ0796f/zYMBKDZ5EWqjrlwYm8s3VIS8n7eiyO7EBFyQ7RW/LVPq43P7umL177ejrlrdul63iu6pGFV0QHMK1D3+A5ptStM7LNK2LqDiKgZEb1KRO/5lMUR0VQiep6IbgxXXcIpAs9BYREs5l3TveKKrFb9ia7rmYEbYyhF1FfeXeqX7FBSr2bVkGNWrjjSdBLqmJ5s2zIP4ZDVIAnTRnU0lOU0oX8LdG6crLjJUqPaibhT44KOZlAVBIhoNhGVENFGv/IcItpGRIVElBvsGMxcxMzj/IpHAkgDcBZAsZaKi8hkd8zzDbpTr+xgaCmKSNauUW1L0haFtRrXrY6PJvRW3Df6m0mXBl0DyypqWwJzAFTIUyIiF4CZAIYCaAtgDBG1JaIORPSJ30+gjq1WAFYy870Abtf3EkQkqFUt/M37Vp7ZmnV8vlDRPGlHCDuo+uYycz4RZfoV9wBQyMxFAEBE8wCMZOZpAEaofP5iAGc8/y9T+RgRgd67rReW/1iKU2fdb6P/fsFWeGh4Wwzr0FDzLFkhrBKNQ39GxgTSAPiOcBR7yhQRUT0ieglAFyKa5Cn+AMAQInoeQH6Ax40norVEtLa0tNRAdYWVMuvXwI29Mn9fQ0XDt0HvuEmV+Dj0lJz0qPXoiLaYc1N3u6thCiPtz8x61SssR+1dvK+rjpV49TDShld63QG/+sx8AMBtfmUnAPiPE/g/bhaAWQCQnZ0ddYF2bK9MzFiyzdIlcSOJ3d0xThqIf3Vsd7zxzY6oHRu4uY/M3geALx+ouE/Df2+7CKNnrcJfgizfYSYjQaAYgG+aRzqAPcaqE3sm9G+BCQbWcxH6qFkKOdr1b5Ualk3iP7mzD77YGjr1U5ijTcNa+P6xwWF7PiPdQQUAWhJRUyKqAmA0gIXmVEvEui/v71e+hrsV/n5VR8uO7TTt02rjziheBVQEpzZFdC6AlQBaEVExEY1j5nMAJgJYAmALgPnMvMm6qopYklm/BprWc6/zY2YXkpO6g4Qwg9rsoDEByhcBUDedMEJFyzaQ0SbqBm+EMMC7RlhCFA7QxH7HaRCpSVXx9B862V2NmKInO0iIaPfwiLZoWLuaquWhI42jg0CoxbOEtbxxwoouHDO2TxRCrVqJCbgnSpfMcORS0he3dG/Hd/+Q8E/RFr/zthbMXUBOArsQWjgyCHjXtq/lkNx9O8iVuBDRwZFBQFjH7u0EW1/gXk8o1PaKQgg3R44JpNZyr6FevWpsrjAZLaxoLdzRvwV6t6xv7l4QQsQwRwaBR0e0Q3aTurhI1p2JEOa1HlxxJAFACA0c2R1UrYoLV3VLt73rIhZdkuUedB/RMfiGJUKIyODIloCwTovUJEM7LwkhwsuRLQERGWRCmRD2kyAgbCe9ckLYR4KAEEI4mAQBYRvvsh3eyXtCiPCTgWFhm1Fd07F9/3FMHCCb7ghhFwkCwjZV4uMwaVgbu6shhKNJO1wIIRxMgoAQQjiYBAEhhHAwCQJCCOFgEgSEEMLBJAgIIYSDSRAQQggHkyAghBAORhxFSzkSUSmAX+yuh0H1Aey3uxJh4qTXCjjr9TrptQLR/3qbMHOK0g1RFQRiARGtZeZsu+sRDk56rYCzXq+TXisQ269XuoOEEMLBJAgIIYSDSRAIv1l2VyCMnPRaAWe9Xie9ViCGX6+MCQghhINJS0AIIRxMgoBFiOhqItpEROeJKNvvtklEVEhE24hoiE95NyLa4LntOaLo3H2XiB4not1EtN7zM8znNsXXHs2IKMfzegqJKNfu+liBiHZ4PpvriWitp6wuEX1ORD95/q1jdz31IKLZRFRCRBt9ygK+tlj7DEsQsM5GAKMA5PsWElFbAKMBtAOQA+BFInJ5bv43gPEAWnp+csJWW/M9w8ydPT+LgJCvPSp56j8TwFAAbQGM8bzOWNTf8356L2pyASxj5pYAlnl+j0ZzUPm7pvjaYvEzLEHAIsy8hZm3Kdw0EsA8Zj7NzNsBFALoQUQNAdRi5pXsHqh5A8AV4atxWCi+dpvrZFQPAIXMXMTMZwDMg/t1OsFIAK97/v86ovTzysz5AH7zKw702mLuMyxBIPzSAOzy+b3YU5bm+b9/ebSaSEQ/eJra3qZ0oNcezWLxNSlhAJ8R0bdENN5T1oCZ9wKA599U22pnvkCvLebeb9lj2AAiWgrgAoWbHmLmBYEeplDGQcojUrDXDne31hS46z8FwD8A3Iwoe40qxeJrUtKbmfcQUSqAz4loq90VsknMvd8SBAxg5oE6HlYMoLHP7+kA9njK0xXKI5La105ELwP4xPNroNcezWLxNVXCzHs8/5YQ0Ydwd4H8SkQNmXmvpzuzxNZKmivQa4u591u6g8JvIYDRRFSViJrCPQC8xtPkPEpEF3qygm4AEKg1EdE8XxqvK+EeJAcCvPZw189kBQBaElFTIqoC96DhQpvrZCoiqkFESd7/AxgM93u6EMCNnrvdiCj9vAYQ6LXF3GdYWgIWIaIrATwPIAVAHhGtZ+YhzLyJiOYD2AzgHIAJzFzmedjtcGcqVAPwqecnGj1NRJ3hbibvAHArAIR47VGJmc8R0UQASwC4AMxm5k02V8tsDQB86MlYjgfwDjMvJqICAPOJaByAnQCutrGOuhHRXAD9ANQnomIAjwGYDoXXFoufYZkxLIQQDibdQUII4WASBIQQwsEkCAghhINJEBBCCAeTICCEEA4mQUAIIRxMgoAQQjiYBAEhhHCw/wcTUwDusnYnkAAAAABJRU5ErkJggg==\n",
-      "text/plain": [
-       "<Figure size 432x288 with 1 Axes>"
-      ]
-     },
-     "metadata": {
-      "needs_background": "light"
-     },
-     "output_type": "display_data"
-    }
-   ],
+   "outputs": [],
    "source": [
     "f, Pxx_den = cusignal.periodogram(gpu_signal, sdr.sample_rate, window='hamm', scaling='spectrum', return_onesided=False)\n",
     "plt.semilogy(cp.asnumpy(cp.fft.fftshift(f/1e4)), cp.asnumpy(cp.fft.fftshift(Pxx_den)))\n",
@@ -239,7 +200,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
@@ -273,24 +234,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "application/vnd.jupyter.widget-view+json": {
-       "model_id": "5a8a10607dae49119771a080ced57589",
-       "version_major": 2,
-       "version_minor": 0
-      },
-      "text/plain": [
-       "Transcribing:   0%|          | 0/1 [00:00<?, ?it/s]"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
-    }
-   ],
+   "outputs": [],
    "source": [
     "transcription = quartznet.transcribe([\"demod_sig.wav\"])"
    ]
@@ -304,17 +250,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "community closed as one as opened as one we didn't have the same in britain um we haven't quite come together as a community in that way so i'm very much in\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "print(transcription[0])"
    ]
@@ -337,24 +275,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "application/vnd.jupyter.widget-view+json": {
-       "model_id": "85d54f7a6efb45e5a8236342905042ae",
-       "version_major": 2,
-       "version_minor": 0
-      },
-      "text/plain": [
-       "Transcribing:   0%|          | 0/1 [00:00<?, ?it/s]"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
-    }
-   ],
+   "outputs": [],
    "source": [
     "transcription = citrinet.transcribe([\"demod_sig.wav\"])"
    ]
@@ -368,17 +291,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 13,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "community closed as one as opened as one we didn't have the same in britain um we haven't quite come together as a community in that way so i'm very much in\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "print(transcription[0])"
    ]
@@ -404,7 +319,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
@@ -420,17 +335,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 18,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "[\"communauté fermée comme une aussi ouverte que celle que nous n'avions pas la même en Grande-Bretagne euh nous n'avons pas tout à fait réuni comme une communauté de cette façon donc je suis très dedans\"]\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "print(translations[0])"
    ]
@@ -452,7 +359,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.12"
+   "version": "3.9.7"
   }
  },
  "nbformat": 4,

--- a/notebooks/sdr/sdr_wfm_demod.ipynb
+++ b/notebooks/sdr/sdr_wfm_demod.ipynb
@@ -4,7 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### [First-Run Only] Environment Setup"
+    "#### [First-Run Only] Environment Setup\n",
+    "\n",
+    "The use of this notebook requires an SDR device to be connected to the local machine."
    ]
   },
   {

--- a/notebooks/sdr/sdr_wfm_demod.ipynb
+++ b/notebooks/sdr/sdr_wfm_demod.ipynb
@@ -1,8 +1,38 @@
 {
  "cells": [
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### [First-Run Only] Environment Setup"
+   ]
+  },
+  {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# SoapySDR module corresponds to SDR selection, e.g.\n",
+    "# soapy-module-lms7 -> lime\n",
+    "# soapy-module-rtlsdr -> rtlsdr\n",
+    "\n",
+    "!mamba install -y soapysdr-module-lms7 pyaudio"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Import Dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
@@ -24,7 +54,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
@@ -43,7 +73,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
@@ -62,7 +92,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
@@ -79,7 +109,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
@@ -104,20 +134,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "<function _signal.default_int_handler>"
-      ]
-     },
-     "execution_count": 6,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "def signal_handler(signum, frame):\n",
     "    stream.stop_stream()\n",
@@ -157,18 +176,11 @@
    "metadata": {},
    "outputs": [],
    "source": []
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
    "language": "python",
    "name": "python3"
   },
@@ -182,9 +194,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.5"
+   "version": "3.9.7"
   }
  },
  "nbformat": 4,
- "nbformat_minor": 2
+ "nbformat_minor": 4
 }

